### PR TITLE
Add uncertainties roles

### PR DESCRIPTION
**Pull request by eteq**
Once v3.1 is released we should merge this since it adds the uncertainties subpackage.

@astrofrog and @mhvk - I added you as deputies given your heavy involvement in the dev. But if you don't want me to do that (or really want to be lead) feel free to say so!

### CODE REVIEW

**Comment by mhvk** [comment]
I do want my `Variable` class there at some point, so happy to be sub-deputy.
(Aside: that source file is damn hard to parse!)

**Comment by adrn** [comment]
As someone who just made other edits to the roles file, I agree with @mhvk -- very hard to parse and edit!

**Comment by eteq** [comment]
@adrn @mhvk - if you think this is bad you should have tried to update it in the pre-js version! :wink: But your point is well taken. It might be just a "use whitespace-and-linebreaks-effectively" solution?
